README.md

```
@@ -52,42 +53,105 @@ sudo apt update
 sudo apt -y install armbian-config
 armbian-config
 ```
+
 </details>
 
-### Contribute
+## Repository Layout
 
-Want to expand **Armbian Config** with new features or tools? Whether you're adding a new software title, enhancing an existing configuration module, or introducing entirely new functionality, we welcome your ideas and code.
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)